src/js/controller/controller.js

```diff
@@ -606,14 +604,18 @@ Object.assign(Controller.prototype, {
         }
 
         function _seek(pos, meta) {
-            if (_model.get('state') === STATE_ERROR) {
+            const state = _model.get('state');
+            if (state === STATE_ERROR) {
                 return;
             }
             _programController.position = pos;
-            if (!_model.get('scrubbing') && _model.get('state') !== STATE_PLAYING) {
+            if (!_model.get('scrubbing') && state !== STATE_PLAYING) {
                 meta = meta || {};
                 meta.startTime = pos;
-                this.play(meta);
+                if (_model.get('streamType') === 'VOD' && (state === STATE_COMPLETE || _model.get('mediaModel').get('duration') - pos < 1)) {
+                    // Only resume VOD playback when  state is complete or position is within 1 second of duration.
```

Comment:
Why not just check the state? Isn't `state === STATE_COMPLETE` enough?

The player never reaches `STATE_COMPLETE` for Live and DVR streams, and never would be complete before the end of a stream,  so this extra logic is not needed.